wordle-web/src/scripts/wordleGame.ts

```diff
@@ -57,4 +57,40 @@ export class WordleGame {
       // The game is over
     }
   }
+
+  getAvailableWords() {
+    const availableWords: string[] = []
+    const secretWordChars = this.secretWord.split('')
```

Comment:
A good start to refactor this to work correctly would be to check against guesses rather than the secret word.